contracts/DAIInterestRateModelV2.sol

```diff
@@ -0,0 +1,113 @@
+pragma solidity ^0.5.16;
+
+import "./JumpRateModel.sol";
+import "./SafeMath.sol";
+
+/**
+  * @title Compound's DAIInterestRateModel Contract (version 2)
+  * @author Compound
```

Comment:
(modified by Dharma)?

Reply:
On that note, even if we want to rename the contract, maybe we should keep it in the same file? It's pretty hard to see the changes directly considering git considers this a full file deletion and file addition?

Reply:
If I'm not mistaken, taking out all the little tweaks fixed that 👍